src/main/Extensions/EmptyTrash/EmptyTrashExtension.ts

```diff
@@ -0,0 +1,116 @@
+import type { AssetPathResolver } from "@Core/AssetPathResolver";
+import type { Extension } from "@Core/Extension";
+import { type FluentIcon, type SearchResultItem } from "@common/Core";
```

Comment:
You can simplify this import:

```suggestion
import type { FluentIcon, SearchResultItem } from "@common/Core";
```